docs/UserGuide.md

```diff
@@ -224,9 +224,11 @@ In addition, there are some _optional_ extra parameters you can use to customize
   Example:`-since 21/10/2017`
 * **`-formats LIST_OF_FORMATS`**: A space-separated list of file extensions that should be included in the analysis. Default: `adoc cs css fxml gradle html java js json jsp md py tag xml`<br>
   Example:`-formats css fxml gradle`
+* **`-isac/--ignore-standalone-config`**: A flag to ignore the standalone config file in the repo (`-isac` as alias). This flag will not overwrite the `Ignore standalone config` field in the csv config file. Default: the standalone config file is not ignored.<br>
```

Comment:
For documentations, let's follow the common style:
`-a, --all`

Reply:
@eugenepeh 

With reference to #390, for confirmation that `--isac` flag does not need to ignore StandaloneConfig of repos in `.csv config files`.

`i.e.` `java -jar RepoSense.jar -config . --isac` does not do anything. It still use StandaloneConfig of remote repos.